### ISSUE TITLE
Sourcing for abstracts may need to be updated to reflect new schema

### ISSUE THREAD

**Issue by wlpotter** [wlpotter/csv-to-srophe#59]
Need first to verify the preferred practice. I am currently getting a validation error on the following code:

```
<note type="abstract" xml:id="abstract2538-1" xml:lang="en">
  <quote source="#bib2538-2">Aeneas was a presbyter and monk</quote>
</note>
```
(cf. #51 for the xml:id issue)

The validation error on `@xml:id` provides the following list of expected attributes: "resp", "source", "target", or "xml:lang". Did we change to have `@source` on the note/desc rather than the nested quote?

Change in persons and places

**Comment by wlpotter** [comment]
@dlschwartz The above is how the transform currently outputs the abstracts with regards to sourcing. I remember sourcing on abstracts being a complicated discussion. Can you remind me where we landed? 

Looking at J of E's persons spreadsheet (and Nathan's comment on the "note abstract.Source 3.en" column especially) this might be a bit more complicated than the previous use cases?

**Comment by dlschwartz** [comment]
@wlpotter If I'm not mistaken, the error is the `@xml:id` and not the sourcing. We allow `@source` and `@resp` on tei:note but we require `@source` on tei:quote. This is because there are instances where we want to source a note differently than we source a quote contained in that note: https://github.com/srophe/syriaca-data/blob/2023.02.21_schemaWork/data/persons/tei/2743.xml#L140.

**Comment by wlpotter** [comment]
Ah, yes that's right. Tested again after removing the xml:id attribute and it appears to be working. Should be resolved now that #51 is fixed
